src/data/types.ts

```diff
@@ -84,17 +84,12 @@ export interface VideoByUserIdResponse {
 }
 
 export interface StreamInfo {
-  game_id: string;
   id: string;
-  language: string;
   started_at: string;
-  tag_ids: string[];
-  thumbnail_url: string;
-  title: string;
   type: string;
-  user_id: string;
-  user_name: string;
-  viewer_count: number;
+  broadcaster_user_id: string;
```

Comment:
Is the removal of all of these properties in this type going to mean we need to fetch the discord data for the announcements separately?

Reply:
I believe so. Some of the data can be read using getLatestVideo.

Confession: I actually updated discord.js as well but that was during the first 2 hours when I was accidentally working on a stale local branch (that had other changes too) and then when I moved to changes manually to a fresh branch I seem to have forgotten the discord changes :(

Reply:
the only problem is that when I go live... this might crash the bot :D

Reply:
I'll take a look after I merge